examples/quick-start-net-core/collector.config.yaml

```diff
@@ -0,0 +1,27 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+processors:
+  batch:
+    timeout: 100ms
+
+  # Data sources: traces
+  probabilistic_sampler:
+    hash_seed: 22
+    sampling_percentage: 100
```

Comment:
Is this section critical? I'd remove it.

Reply:
Done 😄